pandas/core/algorithms.py

```diff
@@ -979,40 +978,32 @@ def value_counts_arraylike(
 
 
 def duplicated(
-    values: ArrayLike, keep: Literal["first", "last", False] = "first"
+    values: ArrayLike,
```

Comment:
looks like the implementation below is just for ndarrays?  can we tighten this here?

Reply:
OK i see the base class EA passes self.  eventually id like to refactor to avoid that, but NBD for now